gef.py

```diff
@@ -2068,8 +2068,11 @@ def gdb_get_nth_previous_instruction_address(addr: int, n: int) -> Optional[int]
 def gdb_get_nth_next_instruction_address(addr: int, n: int) -> int:
     """Return the address (Integer) of the `n`-th instruction after `addr`."""
     # fixed-length ABI
+    if n == 0:
+        raise ValueError(f"`n` must be strictly positive")
```

Comment:
no need for `f"`

Reply:
Couldn't we just return `addr` in this case?

Reply:
^ I guess not, even 1 means first.

Reply:
🤔 maybe it's more a semantic problem: it doesn't make sense to me (even though it's understandable) to say something like "leaving after 0 stop". it's detabable I suppose, but not super critical for this pr

Reply:
I am OK with it being 1. It's not user facing anyhow